tests/fixtures/__recordings__/model-PullRequest_1541417120/should-be-able-to-search_461222074/recording.har

```diff
@@ -0,0 +1,198 @@
+{
+  "log": {
+    "_recordingName": "model: PullRequest/should be able to search",
+    "creator": {
+      "comment": "persister:fs",
+      "name": "Polly.JS",
+      "version": "5.1.0"
+    },
+    "entries": [
+      {
+        "_id": "8f97ffea87c172790edfeba7d54166bc",
+        "_order": 0,
+        "cache": {},
+        "request": {
+          "bodySize": 1224,
+          "cookies": [],
+          "headers": [
+            {
+              "_fromType": "array",
+              "name": "accept",
+              "value": "application/vnd.github.v3+json"
+            },
+            {
+              "_fromType": "array",
+              "name": "user-agent",
+              "value": "octokit-graphql.js/4.6.1 Node.js/14.16.0 (darwin; x64)"
+            },
+            {
+              "_fromType": "array",
+              "name": "authorization",
+              "value": "token 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"
```

Comment:
This token belongs to a "[machine user](https://docs.github.com/en/developers/overview/managing-deploy-keys#machine-users)" (https://github.com/github-metrics-tester) with no private access to any repository.

Reply:
github is unhappy to leak a token (understandably) and it revokes any token that shows up in the diff.

So our polly setup now strips tokens (replacing them with `"token *****"`) before it writes the HAR file to disk.